apps/alpha-test-app/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "alpha-test-app",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vue-tsc && vite build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "sit-onyx": "file:../../packages/sit-onyx",
+    "vue": "^3.4.15",
+    "vue-i18n": "^9.8.0"
+  },
+  "devDependencies": {
+    "@vitejs/plugin-vue": "^5.0.3",
+    "typescript": "^5.3.3",
+    "vite": "^5.0.11",
+    "vue-tsc": "^1.8.27"
```

Comment:
All dependencies except vue-i18n can be removed because we already have them installed in our root package.json

Reply:
you mean only the dev dependencies right? when I remove "vue" vite crashes

Reply:
Yes sorry :)